terminaltables.py

```diff
@@ -22,21 +22,56 @@
 import re
 import struct
 import sys
+import unicodedata
+
+PY2 = sys.version_info[0] == 2
```

Comment:
This is only used (eventually) in _get_width(). Lets move this in there too. I've been doing try/except elsewhere (see ImportError). Let's stick with that for consistency.